.azure-pipelines/Windows-CI.yml

```diff
@@ -52,9 +61,13 @@ jobs:
       git submodule update --init --recursive
       set ONNX_BUILD_TESTS=1
       set ONNX_ML=$(onnx_ml)
-      set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
+      if '$(protobuf_type)' == 'External' (
+        set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
+      ) else (
+        set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=OFF
```

Comment:
@jcwchen One more thing, I found that in Windows Env, when onnx is built with internal protobuf, there will be compile warning treated as error. I use `-DONNX_WERROR=OFF` when internal protobuf is used. Is this acceptable?
![image](https://github.com/onnx/onnx/assets/19584326/6749401a-db97-47bf-a1a3-dd0c90d69009)